src/marqo/core/unstructured_vespa_index/unstructured_vespa_schema.py

```diff
@@ -75,6 +75,14 @@ def _generate_unstructured_schema(cls, marqo_index: UnstructuredMarqoIndex) -> s
             f'+ reduce(query(marqo__add_weights) ' 
             f'* attribute(marqo__score_modifiers), sum)'
         )
+        _score_modifier_expression = (
```

Comment:
Why are you repeating this variable?

Reply:
Yes, just pushed a change to remove the old one, it was undeleted from testing